extensions/project-descriptor.md

```diff
@@ -120,51 +160,42 @@ This defines the buildpacks that a platform should use on the repo.
 
 Either an `id` or a `uri` MUST be included, but MUST NOT include both. If `uri` is provided, `version` MUST NOT be allowed.
 
-## `[[build.env]]`
+#### `[[io.buildpacks.build.env]]`
 
 Used to set environment variables at build time, for example:
 
 ```toml
-[[build.env]]
+[[io.buildpacks.build.env]]
 name = "JAVA_OPTS"
 value = "-Xmx1g"
 ```
 
-## `[metadata]`
-
-This table includes a some defined keys, but additional keys are not validated. It can be used to add platform specific metadata. For example:
-
-```toml
-[metadata.heroku]
-pipeline = "foobar"
-```
-
 ## Example
 
 ```toml
-[project]
+[_]
 id = "io.buildpacks.my-app"
 version = "0.1"
 
-[build]
+[io.buildpacks.build]
 include = [
     "cmd/",
     "go.mod",
     "go.sum",
     "*.go"
 ]
 
-[[build.buildpacks]]
+[_.metadata]
+foo = "bar"
+
+[_.metadata.fizz]
+buzz = ["a", "b", "c"]
+
+[[io.buildpacks.build.buildpacks]]
 id = "io.buildpacks/java"
 version = "1.0"
 
-[[build.buildpacks]]
+[[io.buildpacks.build.buildpacks]]
 id = "io.buildpacks/nodejs"
 version = "1.0"
```

Comment:
I think we need to group the `io.buildpacks.x` tables together to follow TOML best practices. From the [toml spec](https://github.com/toml-lang/toml/blob/master/toml.md#table):
```
# VALID BUT DISCOURAGED
[fruit.apple]
[animal]
[fruit.orange]
```

Reply:
@ekcasey I didn't know you were such a TOML expert.